### ISSUE TITLE
Release plugin version 3.0.0

### ISSUE THREAD

**Issue by SethTisue** [scala/sbt-scala-module#141]
As per discussion on https://github.com/scala/sbt-scala-module/pull/138, the next version is 3.0.0, because newer MiMa and because no more sbt-travisci

I'll start with one or more RCs, which I'll then test in a downstream repo or two.

We've switched from Travis-CI to GitHub Actions in this repo itself, so we're testing that, too.

**Comment by SethTisue** [comment]
tagged 3.0.0-RC1 but it seems I got something wrong, https://github.com/scala/sbt-scala-module/runs/3475141896 has

```
[info] gpg: error checking usability status of FFFFFFFF
[info] gpg: key FFFFFFFF: secret key without public key - skipped
[info] gpg: no default secret key: secret key not available
[info] gpg: signing failed: secret key not available
[error] java.lang.RuntimeException: Failure running 'gpg --batch --passphrase *** --detach-sign --armor --use-agent --output /home/runner/work/sbt-scala-module/sbt-scala-module/target/scala-2.12/sbt-1.0/sbt-scala-module-3.0.0-RC1.pom.asc /home/runner/work/sbt-scala-module/sbt-scala-module/target/scala-2.12/sbt-1.0/sbt-scala-module-3.0.0-RC1.pom'.  Exit code: 2
```

**Comment by SethTisue** [comment]
c4a7634b229f5389e50142c4045e5ce75096a426 and dcb92088a7571ee62c2193ec393403969be3706c seem to have fixed it; https://github.com/scala/sbt-scala-module/actions/runs/1186933937 published successfully

now to actually try it out downstream

**Comment by SethTisue** [comment]
I tried 3.0.0-RC3 out in some downstream repos and that went fine, so I went ahead and re-released it as 3.0.0
